components/filter/demo/filter-search-demo.js

```diff
@@ -96,5 +104,24 @@ class FilterSearchDemo extends LitElement {
 
 	}
 
+	_performTwoListsSearch(searchValue) {
+		const keysToDisplay = [];
+		if (searchValue === '') {
+			keysToDisplay.push('admin', 'instructor');
```

Comment:
```suggestion
	_performInitialSubsetSearch(searchValue) {
		const initialSubset= ['admin', 'instructor'];
		const keysToDisplay = [];
		if (searchValue === '') {
			keysToDisplay = initialSubset;
```

Minor: Probably overkill, but might make it a little clearer how this works. And maybe we go with some naming like "Event on Search - Initial Subset" or something? "Two Lists" makes me think of the header list versus the selected list, even though we _didn't_ go with two lists 😛